### ISSUE TITLE
Can we get a macOS arm64 build?

### ISSUE THREAD

**Issue by skyzyx** [koalaman/shellcheck#2513]
Can we get an `arm64` build for macOS? Or even better, a Universal Binary (possible with the latest GoReleaser — which doesn't require Go)?

**Comment by skyzyx** [comment]
Right. I support users for whom Homebrew isn't an option, and was hoping that the vendor (this repo) could add it to their build matrix so that I don't have to do a custom build.

**Comment by koalaman** [comment]
Unfortunately this is much easier said than done. I try and fail once a year or so, mostly blocked by GHC cross-compilation and the Apple license agreements. So far M1 builds have only been viable on Apple hardware, and requiring access to certain hardware to do a build or release was a huge pain point before 0.7.2 that I'm not about to reintroduce.

Closing as duplicate of #2109

**Comment by koalaman** [comment]
I don't really see how GoReleaser would work, but if you have a proof-of-concept of it building Haskell for aarch64-apple-darwin I'm all ears.
